pkgs/development/tools/gox/default.nix

```diff
@@ -1,21 +1,32 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, go
+}:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "gox";
   version = "1.0.1";
 
-  goPackagePath = "github.com/mitchellh/gox";
-
   src = fetchFromGitHub {
     owner = "mitchellh";
     repo = "gox";
     rev = "v${version}";
     sha256 = "0mkh81hd7kn45dz7b6yhzqsg2mvg1g6pwx89jjigxrnqhyg9vrl7";
   };
+  vendorSha256 = null;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postFixup = ''
+    wrapProgram $out/bin/gox --prefix PATH : ${lib.makeBinPath [ go ]}
```

Comment:
This does not work :(

Reply:
Try allowing `go` reference explicitly (AFAIU that's your intention):

```diff
--- a/pkgs/development/tools/gox/default.nix
+++ b/pkgs/development/tools/gox/default.nix
@@ -19,6 +19,8 @@ buildGoModule rec {

   nativeBuildInputs = [ makeWrapper ];

+  # We are about to embed path to `go` bianry:
+  allowGoReference = true;
   postFixup = ''
     wrapProgram $out/bin/gox --prefix PATH : ${lib.makeBinPath [ go ]}
   '';
```